packages/mdc-ripple/mdc-ripple.scss

```diff
@@ -20,36 +20,5 @@
 // THE SOFTWARE.
 //
 
-@import "@material/animation/functions";
-@import "@material/theme/mixins";
-@import "./common";
```

Comment:
I don't think you want to remove `common` here... all of the emitted keyframes styles disappeared.